plugins/woocommerce/includes/import/class-wc-product-csv-importer.php

```diff
@@ -3,7 +3,7 @@
  * WooCommerce Product CSV importer
  *
  * @package WooCommerce\Import
- * @version 3.1.0
+ * @version x.x.x
```

Comment:
We should add the next version here before merging, because IIRC, WooCore release process doesn't have a step to look for `x.x.x` placeholders and replace later.

Reply:
I thought that the Woorelease updated these automatically: https://github.com/woocommerce/woorelease/blob/b8a2ec2550f6a6664791962e6f9e29dd61d1b61e/includes/tools/class-version-replace.php#L36

and also `trunk` has some files with `x.x.x` right now. 

But updated it to `10.0.0`.